app/controllers/dashboard/README.md

```diff
@@ -6,7 +6,7 @@ Publish an manage internship_offers as well as internship_applications.
 Employer manage their own offers
 Operator can manage their offers as well as offers delegated by employers
 
-## MainTeacher, Other, SchoolManager, Teacher
+## SchoolManagement
```

Comment:
TODO: clarify roles and responsability. school_manager must have an ac-xxxx.xx to register ; is notified of other roles registering.